vitess.io/_layouts/base.liquid

```diff
@@ -12,9 +12,6 @@
     <meta name="og:title" content="{{ site.title }} / {{page.title}}"/>
     <meta name="og:image" content="{{ site.baseurl }}/images/{{ site.logo }}"/>
     <meta name="og:description" content="{{ site.description }}"/>
-
-    <!-- GA -->
-    {% include analytics.liquid %}
```

Comment:
Fair enough ... moved to <head>. I found this stackoverflow thread helpful ... it says that GA used to instruct you to put the snippet at the bottom of the page before they added support for handling partial loading of pages. Now that they do, they tell you to put it in the <head> section. I guess I wasn't up-to-date on the new guidelines.